v3/control.go

```diff
@@ -23,6 +23,8 @@ const (
 
 	// ControlTypeMicrosoftNotification - https://msdn.microsoft.com/en-us/library/aa366983(v=vs.85).aspx
 	ControlTypeMicrosoftNotification = "1.2.840.113556.1.4.528"
+	// ControlTypeSubtreeDelete - https://docs.microsoft.com/zh-cn/previous-versions/windows/desktop/ldap/ldap-server-tree-delete-oid
```

Comment:
May you please use a english version as reference for this control type?

Reply:
I suggest you use: https://datatracker.ietf.org/doc/html/draft-armijo-ldap-treedelete-02